desktop/cmp/grouping/GroupingChooser.ts

```diff
@@ -283,7 +292,7 @@ const addDimensionControl = hoistCmp.factory<GroupingChooserModel>({
                     // ensure the Select loses its internal input state.
                     key: JSON.stringify(options),
                     options,
-                    placeholder: 'Add...',
+                    placeholder: 'Add level...',
```

Comment:
We can drop this addition of "level" - added here and in mobile as it felt just a bit more clear to me, don't feel strongly